### PR TITLE
#7768: Created plugin for CKeditor to change the font color and backg…

### PR DESCRIPTION
**Pull request by aster06**
…round color.

### CODE REVIEW

**Review by ivandudardb — APPROVED** [review]
LGTM

File: web/modules/custom/ckeditor_custom/js/ckeditor5_plugins/buttonPlugin/src/buttonview.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  textDefaultColor
+} from './buttonconfig';
+
+export default class FormView extends View {
+  constructor( locale, componentFactory, defaultColor) {
+    super( locale);
+    this.focusTracker = new FocusTracker();
+    this.keystrokes = new KeystrokeHandler();
+
+    //let textDefaultColor = drupalSettings.ckeditor_custom_button.default_color;
```
Comment (lavrenchuk-dev) [inline web/modules/custom/ckeditor_custom/js/ckeditor5_plugins/buttonPlugin/src/buttonview.js]: ^

File: web/modules/custom/ckeditor_custom/js/ckeditor5_plugins/buttonPlugin/src/buttonview.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+    // Creates Dropdowns for Color, Size, Text color
+    this.colorDropdown = this._createSelectionDropdown(locale, 'Color', colorIcon, 'color', colorOptions, defaultColor)
+    this.sizeDropdown = this._createSelectionDropdown(locale, 'Size', sizeOptions[defaultSize].icon, 'size', sizeOptions, defaultSize )
+    this.textDropdown = this._createSelectionDropdown(locale, 'Text', colorTextIcon, 'text', textColorOptions, textDefaultColor )
+    // Creates the main input field.
+    // this.innerTextInputView = this._createInput( 'Button Text' );
```
Comment (lavrenchuk-dev) [inline web/modules/custom/ckeditor_custom/js/ckeditor5_plugins/buttonPlugin/src/buttonview.js]: ^

File: web/modules/custom/ckeditor_custom/js/ckeditor5_plugins/buttonPlugin/src/buttonview.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    this.cancelButtonView = this._createButton( 'Cancel', icons.cancel, 'ck-button-cancel' );
+
+    // Delegate ButtonView#execute to FormView#cancel.
+    this.cancelButtonView.delegate( 'execute' ).to( this, 'cancel' );
+
+    this.childViews = this.createCollection( [
+      this.sizeDropdown,
+      this.colorDropdown,
+      this.textDropdown,
+      // this.innerTextInputView,
```
Comment (lavrenchuk-dev) [inline web/modules/custom/ckeditor_custom/js/ckeditor5_plugins/buttonPlugin/src/buttonview.js]: ^

**Review by lavrenchuk-dev — CHANGES_REQUESTED** [review]

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**